tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py

```diff
@@ -2,183 +2,135 @@
 
 import great_expectations as gx
 from great_expectations.core.yaml_handler import YAMLHandler
-from great_expectations.datasource.new_datasource import Datasource
+from great_expectations.datasource.fluent.config_str import ConfigStr
+from tests.test_utils import load_data_into_test_database
 
 yaml = YAMLHandler()
 
+"""
```

Comment:
@willfeltman Small note (non-block -- if can be added to a follow-on PR) our style for adding snippets in Python modules has been as follows:
```
# Python
# <snippet name="this_module_repository_path.py my_specific_tag">
... code ...
# </snippet>
```

In other words, the `# Python` comment at the top needs to be there, because it will render in the document and make it clear that this is a `Python` code snippet.

Thanks!
bucket_name = "superconductive-docs-test"
```

Reply:
Could you confirm this? It's my first time hearing about the `#python` change, and I can't seem to find it in some of the more recent documents 

https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context/

I think this is one you updated recently